packages/react-day-picker/src/hooks/useInput/useInput.ts

```diff
@@ -3,26 +3,80 @@ import * as React from 'react';
 import { differenceInCalendarDays, format as _format, parse } from 'date-fns';
 import enUS from 'date-fns/locale/en-US';
 
-import { DayClickEventHandler, MonthChangeEventHandler } from '../../types';
-
 import { parseFromToProps } from '../../contexts/DayPicker/utils';
-
 import {
-  UseInputDayPickerProps,
-  UseInputFieldProps,
-  UseInputOptions
-} from './types';
-import { UseInput } from './types/UseInput';
+  DayClickEventHandler,
+  DayPickerProps,
+  MonthChangeEventHandler
+} from '../../types';
 import { isValidDate } from './utils/isValidDate';
 
-const DefaultFormat = 'PP';
+/** Represent the props to attach to the input field. */
```

Comment:
I did not like the idea of placing a type in each file, so when I met them – I move to the file where they are used.